feat: Add adminKey flag to node setup command

Ivo-Yankov opened the pull request:
## Description

### Related Issues

* Closes #1042

jeromy-cannon reviewed (changes requested):
@Ivo-Yankov ,

A few questions/thoughts:

1. As someone using Solo, how do I provide a different key for each node?  When looking at the code I see that you allow a comma separated list of keys.  However, when looking at the `flags.ts` code, I only see the description: `Admin key`.  I think we need to provide a better description and provide a usage example.
2. Since we don't sign anything, I think it would be more friendly to just supply the public key. It will be smaller, easier to supply on the command line, and will be safer, since the private key isn't really needed here.
3. rename the flag from `--admin-key` to `--admin-public-keys`

Ivo-Yankov commented:
> A few questions/thoughts:
> 
> 1. As someone using Solo, how do I provide a different key for each node?  When looking at the code I see that you allow a comma separated list of keys.  However, when looking at the `flags.ts` code, I only see the description: `Admin key`.  I think we need to provide a better description and provide a usage example.
> 2. Since we don't sign anything, I think it would be more friendly to just supply the public key. It will be smaller, easier to supply on the command line, and will be safer, since the private key isn't really needed here.
> 3. rename the flag from `--admin-key` to `--admin-public-keys`

@jeromy-cannon I addressed points 2 and 3 and I think that mostly covers point 1. But I'm not sure on what is the appropriate way to provide a usage example.

jeromy-cannon commented on src/commands/flags.ts:
```diff
@@ -1351,6 +1351,17 @@ export class Flags {
+    constName: 'adminPublicKeys',
+    name: 'admin-public-keys',
+    definition: {
+      describe: 'Comma separated list of DER encoded ED25519 public keys',
```
```suggestion
      describe: 'Comma separated list of DER encoded ED25519 public keys and must match the order of the node aliases',
```

swirlds-automation commented:
:tada: This PR is included in version 0.35.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@hashgraph/solo/v/0.35.0)
- [GitHub release](https://github.com/hashgraph/solo/releases/tag/v0.35.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

swirlds-automation commented:
:tada: This PR is included in version 0.35.0 :tada:

The release is available on:
- [npm package (@release-0.35.x dist-tag)](https://www.npmjs.com/package/@hashgraph/solo/v/0.35.0)
- [GitHub release](https://github.com/hashgraph/solo/releases/tag/v0.35.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
